quiz.css

```diff
@@ -0,0 +1,69 @@
+/*universal stylings*/
+* {
+  box-sizing: border-box;
+  font-family: Arial, Serif;
+  font-weight: bold;
+}
+
+/*sizing and borders for each article/product*/
+article {
+  width: 30%;
+  display: inline-block;
+  border: 1px solid lightblue;
+  margin: 1.5%;
+}
+
+.productTitle {
+  border: 1px dotted lightgray;
+  text-align: center;
+}
+
+/*resizing and centering product images*/
+img {
+  width: 75%;
+  display: block;
+  margin: auto;
+}
+
+/*styles for the product descriptions*/
+.description {
+  text-align: justify;
+  padding: 0 4%;
+  font-size: 100%;
+}
+
+/*specification and pricing headers*/
+h2 {
+  font-weight: bold;
+  font-size: larger;
+  padding-left: 3%;
+}
+
+/*removing unnecessary pricing header margins*/
+.pricing {
+  margin: 0;
+}
+
+/*available text styles*/
+.avail {
+  color: yellow;
+  background-color: darkgrey;
+  padding: 5px 0 5px 2%;
+}
+
+/*styles for spceifications and pricing list items*/
+ul {
+  list-style: none;
+  font-size: .8em;
+
+}
+
+li {
+  margin: -10px;
+  padding: 0;
+}
+
+/*resizing the specifications valid until statement*/
+footer {
+  font-size: .6em;
+}
```

Comment:
Always include a new line at the end of every file. Consider adding `ensure_newline_at_eof_on_save` to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior